coalib/coala_main.py

```diff
@@ -2,7 +2,6 @@
 import platform
 
 from pyprint.ConsolePrinter import ConsolePrinter
-
```

Comment:
Please reinsert the blank line :) It separates external imports from internal ones